docs/user/configuration.md

```
@@ -0,0 +1,102 @@
+# Configuration
+
+Tanjun offers extensive per-server configuration to tailor the bot to your community's needs.
+
+## General Commands
+
+### Help
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)